src/initialize-custom-elements.ts

```diff
@@ -47,7 +47,9 @@ function assignAttributes(fromElement: Element, toElement: Element): void {
 
 function whenRendered(app, callback) {
   if (app['_rendering']) {
-    setTimeout(whenRendered, 10, app, callback);
+    self.Promise.resolve.then(() => {
```

Comment:
should this be self.Promise.resolve().then(() => {? When I try this in a codepen I get an infinite loop.

Reply:
@code0100fun yes it surely should be.